src/blocks/scratch3_looks.js

```diff
@@ -325,7 +340,7 @@ class Scratch3LooksBlocks {
     }
 
     think (args, util) {
-        this._updateBubble(util.target, 'think', String(args.MESSAGE).substr(0, Scratch3LooksBlocks.SAY_BUBBLE_LIMIT));
+        this.runtime.emit('SAY', util.target, 'think', args.MESSAGE);
```

Comment:
Can we change 'SAY' to a constant variable and rename it to 'SAY_OR_THINK'?

Reply:
As a static getter on `Runtime`, on `Scratch3LooksBlocks`, or just as a `const` at the top of the file?

Reply:
I think static getter on Scratch3LooksBlocks

Reply:
Hopefully [this](https://github.com/LLK/scratch-vm/pull/2187/commits/ffd2a2a9e8f7a95df6bd81d60fe1457fcb4a1918) is good. There are a lot of places in the codebase that still use the raw event string `'SAY'` as the event name, but the codebase also uses raw event strings for many other event listeners as well, so I opted against changing all of the `'SAY'` listeners over to `Scratch3LooksBlocks.SAY_OR_THINK`.

Even if I did rename all the `'SAY'` events in this codebase to `'SAY_OR_THINK'`, `scratch-render`'s tests use `'SAY'` events as well, so those would need to be changed simultaneously to prevent breakage.